k8s/service.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: minio-kv-service
+  labels:
+    app: minio-kv
+spec:
+  ports:
+    - port: 8080
+      name: clientport
+      targetPort : 9000
```

Comment:
Target port looks wrong? why isn't it `8080`?

Reply:
Target port was set based on the docker run command in the README - docker run ... --name minio -p **9000:9000** minio/minio server /tmp/

I can change to 8080 if that's the port your app is listening on.